Formula/numpy.rb

```diff
@@ -4,6 +4,7 @@ class Numpy < Formula
   url "https://files.pythonhosted.org/packages/0a/c8/a62767a6b374a0dfb02d2a0456e5f56a372cdd1689dbc6ffb6bf1ddedbc0/numpy-1.22.1.zip"
   sha256 "e348ccf5bc5235fc405ab19d53bec215bb373300e5523c7b476cc0da8a5e9973"
   license "BSD-3-Clause"
+  revision 1
```

Comment:
A `revision` is not necessary, no? Since this was probably last built with the right version of `setuptools`.

Reply:
Oh, right. :hand_over_mouth: